torrt/trackers/anilibria.py

```diff
@@ -25,14 +25,14 @@ def get_download_link(self, url):
         page_soup = self.get_response(url, as_soup=True)
 
         available_qualities = {}
-        divs = page_soup.select('div.download-torrent')
-        for div in divs:
-            quality_span = div.select('div.torrent-first-col > span')
-            quality_str = quality_span[0].text.strip()
+        rows = page_soup.select('#publicTorrentTable tr')
```

Comment:
I've just added a convinience to automatically test torent extraction https://github.com/idlesign/torrt/commit/b6d83e687056c62fe8dfea672efac98509397f5d

You can add the following or something to test your code automatically:
```
    test_urls = [
        'https://www.anilibria.tv/release/black-clover.html',
    ]
```